pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -686,6 +686,9 @@ func allocatableResources(memory, cpu string) v1.ResourceList {
 
 // DaemonSets should not place onto nodes with insufficient free resource
 func TestInsufficientCapacityNodeDaemonDoesNotLaunchPod(t *testing.T) {
+	if utilfeature.DefaultFeatureGate.Enabled(features.ScheduleDaemonSetPods) {
```

Comment:
We have a test that verifies that DS controller creates pods for all appropriate nodes regardless of their schedulability, when this feature is enabled. Please delete the "skip" at line 458 to enable the test.

Reply:
Sure.